GOVERNANCE.md

```diff
@@ -69,6 +99,49 @@ The Technical Committee meets regularly and the meetings are intended to be open
 Quorum for Technical Committee meetings requires at least a majority of all voting members of the Technical Committee to be present.
 The Technical Committee  may continue to meet if quorum is not met, but will be prevented from making any decisions at the meeting.
 
+### Technical Committee Elections and Voting
+
+Representatives to Eiffel Community Technical Committee are elected by their peers in the Eiffel Community on the basis of merit.
+
+#### Election Rules
+
+Anyone who is a maintainer of one or more projects hosted by Eiffel Community is eligible to run for a seat in Eiffel Technical Committee or nominate another maintainer as a candidate.
+
+Anyone who is a contributor to one or more projects hosted by Eiffel Community is eligible to vote on Technical Committee Elections.
+
+Community elections are overseen by [Election Officers](#Election-Officers).
+
+#### Election Process
+
+The election process consists of 3 phases
+
+* **Preparation Phase**:
+  * Election Officers determine the list of community members who are eligible to **run** in community elections announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * Election Officers determine the list community members who are eligible to **vote** in community elections and announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+* **Nomination Phase**:
+  * Once the electorate and candidates are determined and notified, Election Officers start nomination period for Technical Committee elections by sending an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) Nomination period is limited to **one week**.
+  * After the commencement of the nomination phase, eligible candidates who wish to run for a seat in Technical Committee announce their nomination by sending an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community). (self-nomination) If community members want to nominate someone else to Technical Committee, they can do so by sending nomination mail to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) but this must then be accepted by the nominated community member by responding to their nomination mail in order for their candidacy to be valid. Only maintainers can self-nominate or nominate other maintainers.
+  * Upon the completion of the nomination phase, Election Officers collate the nominations and announce final list of candidates on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * Announcement of candidates ends the nomination phase.
+* **Election Phase**:
+  * Eiffel Community uses Condorcet Internet Voting Service System ([CIVS](https://civs.cs.cornell.edu/)) to hold elections. The instructions to prepare the election poll is available [here](https://wiki.opendaylight.org/display/ODL/CIVS+Voting+System+Information).
```

Comment:
I propose to update this and leave the determination of election medium to election officers since I'm not sure if CIVS can be used via sending mail to a maillist.

Survey Monkey seems to be much simpler: https://help.surveymonkey.com/articles/en_US/kb/Can-I-create-a-poll-or-voting-environment

I'll update Election Phase with this statement which we can discuss during the meeting on Thursday.